fixbackend/config.py

```diff
@@ -77,6 +79,8 @@ def parse_args(argv: Optional[Sequence[str]] = None) -> Namespace:
     parser.add_argument("--cdn-bucket", default=os.environ.get("FIXUI_CDN_BUCKET", "fix-ui"))
     parser.add_argument("--fixui-sha", default=os.environ.get("FIXUI_SHA", ""))
     parser.add_argument("--static-assets", type=Path, default=os.environ.get("STATIC_ASSETS"))
+    parser.add_argument("--session-ttl", type=int, default=int(os.environ.get("SESSION_TTL", 3600)))
+    parser.add_argument("--oauth-redirect-url", default=os.environ.get("OAUTH_REDIRECT_PATH", "/"))
```

Comment:
We do this for development purposes, right?
Please add a comment, describing the purpose.

Reply:
noted